docs/en/sql-reference/functions/introspection.md

```diff
@@ -333,10 +324,8 @@ demangle(symbol)
 
 **Returned value**
 
-- Name of the C++ function.
-- Empty string if a symbol is not valid.
-
-Type: [String](../../sql-reference/data-types/string.md).
+- Name of the C++ function. [String](../../sql-reference/data-types/string.md).
```

Comment:
`Name of the C++ function, or an empty string if the symbol is not valid. [String](../../sql-reference/data-types/string.md)